baseConfig.json

```diff
@@ -181,6 +181,8 @@
         "no-magic-numbers": "off",
         "no-mixed-operators": "off",
         "array-bracket-spacing": [ "error", "always" ],
+        "array-bracket-newline": "off",
+        "array-element-newline": [ "error", { "multiline": true } ],
```

Comment:
If an array literal spans over multiple lines than this rule requires each **element** of the array to be on a separate line.

Examples:

```js
// good
foo = [ 1, 2, 3, 4 ];
// good
foo = [
  1,
  2,
  3,
  4
];
// bad
foo = [
  1, 2, 3, 4
];
```